src/components/TryExtensionLinkDialog.tsx

```diff
@@ -100,11 +100,30 @@ export const TryExtensionLinkDialog: FunctionComponent<{
 const TryExtensionBox: FunctionComponent<{
   onInstallClick: () => void
 }> = props => {
+  const [isClicked, setIsClicked] = useState(false)
+
+  const clickHandler = () => {
+    if (isClicked) {
+      window.location.reload()
+    } else {
+      props.onInstallClick()
+      setIsClicked(true)
+    }
+  }
+
   return (
     <div class="-cbwsdk-extension-dialog-box-top">
       <div class="-cbwsdk-extension-dialog-box-top-install-region">
         <h2>Try the Coinbase Wallet extension</h2>
-        <button onClick={props.onInstallClick}>Install</button>
+        {isClicked && (
+          <div class="-cbwsdk-extension-dialog-box-top-subtext">
+            After installing Coinbase Wallet, refresh the page and connect
+            again.
+          </div>
+        )}
+        <button onClick={clickHandler}>
```

Comment:
```suggestion
        <button type="button" onClick={clickHandler}>
```
all button elements should always have an explicit type, because the default type is `submit`